mscore/exportxml.cpp

```diff
@@ -3660,7 +3659,7 @@ void ExportMusicXml::textLine(TextLine const* const tl, int staff, const Fractio
                   }
             else
                   lineEnd = "down";
-            rest += QString(" end-length=\"%1\"").arg(hookHeight * 10);
+            rest += QString(" end-length=\"%1\"").arg(hookHeight * TEN);
```

Comment:
I don't like using TEN constant here and there. I would create a method which encapsulates the meaning of this operation. If the operation relates to musicXmlExport only, let's call it `convertValueToMXLExportSemantic`. The method will multiply values by 10. In this case we won't need the global constant value. Also, if we decide to refactor the code, all the changes will be obvious.

My general point is that the code should be clear. Using TEN instead of 10 doesn't make code clearer. The semantic of the TEN should be highlighted and ideally encapsulated.

Reply:
I see your point.  I'll make the change.